docs/tutorial/tutorial_basics.rst

```diff
@@ -324,15 +324,18 @@ the data at the :py:class:`pysat.Instrument` level that behaves the same whether
     # Convenient data assignment
     dmsp['ti'] = new_array
 
-    # Convenient data broadcasting assignment, sets a single value to all times
-    dmsp['ti'] = single_value
-
     # Assignment through index slicing
     dmsp[0:10, 'ti'] = sub_array
 
     # Assignment through datetime slicing
     dmsp[start:stop, 'ti'] = sub_array
 
+Note that a :py:class:`pandas.DataFrame` will hold to a strict time index, meaning that
+all variables must be the same length.  Assigning a single value to an array
+will broadcast that value over all coordinates, resulting in a constant value
+as a function of time. To allow for multiple dimensions, the :py:class:`xarray.Dataset`
+allows more flexibility.  If a single value is assigned to a new variable, it
+will be sent to the coords.
```

Comment:
```suggestion
Note that a :py:class:`pandas.DataFrame` will hold to a strict time index, 
meaning that all variables must be the same length.  Assigning a single 
value to an array will broadcast that value over all coordinates, resulting 
in a constant value as a function of time. To allow for multiple dimensions, 
the :py:class:`xarray.Dataset` allows more flexibility.  If a single value is 
assigned to a new variable, it will be sent to the coords.
```